### ISSUE TITLE
Save issues

### ISSUE THREAD

**Issue by evacfk** [Kwoth/NadekoBot#1356]
Load presents this error:

`07:02:27 CommandHandler | Command Errored after 0/0/0.06s
        User: evac#7829 [93237270869479424]
        Server: evac's wonderland [93284396739604480]
        Channel: music-bot [206105123984769024]
        Message: .load blackpink
        Error: Failed to parse Int32 ` 

When I .pl and attempt to save, the music will cut out/stop, errors spam across the terminal until it eventually crashes and restarts the bot

```6:57:31 MusicService | No song found. Data less than 6
06:57:31 MusicService | No song found. Data less than 6
06:57:37 MusicService | No song found. Data less than 6
06:57:37 MusicService | No song found. Data less than 6
06:57:37 MusicService | No song found. Data less than 6
06:57:37 MusicService | No song found. Data less than 6
06:57:37 MusicService | No song found. Data less than 6
06:57:37 MusicService | No song found. Data less than 6
06:57:37 MusicPlayer | Song Canceled
06:57:39 MusicService | No song found. Data less than 6
06:57:44 MusicService | No song found. Data less than 6
06:57:48 MusicPlayer | Next song
06:57:49 MusicPlayer | Starting
06:57:50 MusicService | No song found. Data less than 6
06:57:55 MusicService | No song found. Data less than 6
06:58:01 MusicService | No song found. Data less than 6
06:58:01 MusicService | No song found. Data less than 6
06:58:03 MusicService | No song found. Data less than 6
06:58:03 MusicService | No song found. Data less than 6
06:58:03 MusicService | No song found. Data less than 6

Unhandled Exception: 06:58:08 MusicService | No song found. Data less than 6
06:58:10 MusicService | No song found. Data less than 6
06:58:10 MusicService | No song found. Data less than 6
06:58:13 MusicService | No song found. Data less than 6
06:58:18 MusicService | No song found. Data less than 6
06:58:23 MusicService | No song found. Data less than 6
06:58:29 MusicService | No song found. Data less than 6
06:58:36 MusicService | No song found. Data less than 6
06:58:45 MusicService | No song found. Data less than 6
06:58:53 MusicService | No song found. Data less than 6
06:59:06 MusicService | No song found. Data less than 6
06:59:11 MusicService | No song found. Data less than 6
06:59:16 MusicService | No song found. Data less than 6
06:59:19 MusicService | No song found. Data less than 6
06:59:24 MusicService | No song found. Data less than 6
06:59:30 MusicService | No song found. Data less than 6
06:59:41 MusicService | No song found. Data less than 6
06:59:42 MusicService | No song found. Data less than 6
06:59:50 MusicService | No song found. Data less than 6
06:59:58 MusicService | No song found. Data less than 6
07:00:10 MusicService | No song found. Data less than 6
System.ComponentModel.Win32Exception: Cannot allocate memory
   at Interop.Sys.ForkAndExecProcess(String filename, String[] argv, String[] envp, String cwd, Boolean redirectStdin, Boolean redirectStdout, Boolean redirectStderr, Int32& lpChildPid, Int32& stdinFd, Int32& stdoutFd, Int32& stderrFd)
   at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at NadekoBot.Services.Impl.YtdlOperation.<GetDataAsync>d__2.MoveNext() in /root/NadekoBot/src/NadekoBot/Services/Impl/Ytdl.cs:line 32
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NadekoBot.Services.Music.MusicService.<>c__DisplayClass26_0.<<GetYoutubeVideo>b__0>d.MoveNext() in /root/NadekoBot/src/NadekoBot/Services/Music/MusicService.cs:line 354
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NadekoBot.Services.Music.MusicPlayer.<PlayerLoop>d__98.MoveNext() in /root/NadekoBot/src/NadekoBot/Services/Music/MusicPlayer.cs:line 167
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
```

**Comment by Kwoth** [comment]
Load is working properly, you need to specify the ID, not the name of the playlist. Just tested and it works.

However that second error shows a pretty big bug, I'll try to resolve it right away

**Comment by Kwoth** [comment]
just fixed, wait for 1.55 soon:tm:
